components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -2849,28 +2849,19 @@ public static NativeObject jsFunction_getSubscriber(Context cx, Scriptable thisO
     }
 
     public static boolean jsFunction_addSubscriber(Context cx, Scriptable thisObj, Object[] args, Function funObj)
-            throws ScriptException, APIManagementException, UserStoreException {
+            throws ScriptException, APIManagementException{
 
         if (args != null && isStringArray(args)) {
-            Subscriber subscriber = new Subscriber((String) args[0]);
+            String username = (String) args[0];
```

Comment:
Fixed in #1581 (https://github.com/wso2/carbon-apimgt/commit/eb1deba65015d972478713ccf5557bc57be73612)